### ISSUE TITLE
Vanishing items on Darrowshire

### ISSUE THREAD

**Issue by pvtmakaveli** [elysium-project/server#2584]
Yesterday, on 15.09.2017, I've bought a [Warmonger] on Auction House. After that I covered it with [Red Ribboned Wrapping Paper]. When I opened my [Red Ribboned Gift] it was empty. Sword, which costs 50 gold just disappeared. 

After that I've made an in-game ticket with the description of my problem. GM Spionida confirmed that there was a bug and asked me to report this here.

I'm pleased to be answered so fast. Thank You for that, but I still don't have either gold or sword. Please assist me to get my sword or my money back. Thank You.

Маkiaveli (56 NE rogue), Darrowshire.

**Comment by Spionida** [comment]
Tested recently on several game characters. Reproducible.

**Comment by GuybrushGit** [comment]
After unwrapping the item shows up for me, but after relogging the item goes missing. 
Is this what happened?

**Comment by Spionida** [comment]
No, a mailed wrapped item, when opened is empty, AND/OR it disappears before the logout. Varies.

**Comment by ewilhelml** [comment]
I've had an item vanish as well. Through the mailbox. It was Stockade Pauldrons which had sold for 220g. No fix for this either.
